scalability_and_performance/recommended-cluster-scaling-practices.adoc

```diff
@@ -0,0 +1,15 @@
+[id="recommended-cluster-scaling-practices"]
+= Recommended cluster scaling practices
+include::modules/common-attributes.adoc[]
+:context:
+
+toc::[]
+
+This topic provides recommended best practices for scaling {product-title}
+clusters.
+
+The cluster can be scaled by increasing or decreasing the replicas in the worker
```

Comment:
I'd rewrite to make this sentence active. I'd also swap this sentence and the previous one. Or say something like "Apply the following best practices to scale the number of worker machines in your {product-title} cluster. You scale the worker machines by increasing or decreasing the number of replicas that are defined in the worker MachineSet."